api/end_device_services.proto

```diff
@@ -61,3 +61,20 @@ service EndDeviceRegistry {
     };
   };
 }
+
+service EndDeviceTemplateConverter {
+  // Returns the configured formats to convert from.
+  rpc ListFormats(google.protobuf.Empty) returns (EndDeviceTemplateFormats) {
+    option (google.api.http) = {
+      get: "/edtc/formats"
```

Comment:
Nitpick: Settle for either `etdc` or `dtc`, but avoid mixing.

Reply:
We consistently use `EndDevice` in our protos/API, including gRPC gateway paths, except within "applications" but those are always end devices.

Also, the `DeviceTemplateConverter` **component** may also expose the `GatewayTemplateConverter` **service** in the future; a gateway is also a device.